app/models/user.rb

```diff
@@ -45,6 +45,10 @@ class User < ApplicationRecord
   acts_as_followable
   acts_as_follower
 
+  has_many :authored_user_subscriptions, class_name: "UserSubscription", foreign_key: :author_id, inverse_of: :author, dependent: :destroy
```

Comment:
The `UserSubscriptions` the `User` is an `Author` of.

Reply:
This took me a moment to untangle. You mean the user is the author of the article that another user subscribed to, right?

Reply:
That's correct! In the future, it could be the author of the `Comment` or something else. It's the author of the source.

Reply:
Do you think it would be better if I did `:source_authored_user_subscriptions`?